figo.net/FigoSession.cs

```diff
@@ -29,49 +29,68 @@ public class FigoSession {
         /// The timeout for a API request
         /// </summary>
         private int _timeout = 5000;
-        public int Timeout {
```

Comment:
it would be great if you could re-submit this without formatting changes to keep the diff clean